app/views/posts/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>These are the Posts of the user!</h1>
+<section class="container">
+<article class="user-card">
+<img src="<%= @user.photo %>" class="user-photo" alt="user photo">
+<div class="user-card-info">
+ <h2><%= @user.name %></h2>
+  <p>Number of posts: <%= @user.posts_count %></p>
+</div>
+</article>
+   <% @posts.each do |post| %>
+        <%= render partial: "post_card", locals: { post: post, comments: post.recent_comments } %>
+    <% end %>
+</section>
```

Comment:
- [ ] Great job on implementing this view, however NOTE that this view should follow the given wireframe in terms of the layout of elements.

Your design
![image](https://github.com/mariegrace31/blogApp/assets/81932478/cea3045e-9a8d-4b1d-acf0-ea61544beb24)

Given frame
![image](https://github.com/mariegrace31/blogApp/assets/81932478/c38b7234-a41b-4849-a1d8-d685364d372a)


#### Pay attention to the layout of elements as they don't match the given design